### PR TITLE
LPS-157078 Clean up KB - Asset renderer and display contexts

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -500,6 +491,52 @@ private String _getTemplatePath() {
...
+				_themeDisplay.getScopeGroupId(), parentResourcePrimKey,
+				KBActionKeys.ADD_KB_ARTICLE);
+		}
+
+		return AdminPermission.contains(
+			_themeDisplay.getPermissionChecker(),
+			_themeDisplay.getScopeGroupId(), KBActionKeys.ADD_KB_ARTICLE);
+	}
+
+	private boolean _hasAddKBFolderPermission() throws PortalException {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java:518]: We are adding 2 times the folder on the creation menu

image

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java [hunk 2]
```diff
@@ -150,31 +144,32 @@ public CreationMenu getCreationMenu() throws PortalException {
...
+							PortalUtil.getClassNameId(
+								KBFolderConstants.getClassName())
+						).setParameter(
+							"parentResourcePrimKey", parentResourcePrimKey
+						).buildPortletURL());
 
+					dropdownItem.setLabel(
+						LanguageUtil.get(_httpServletRequest, "folder"));
+				});
 			creationMenu.addDropdownItem(
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java:173]: This code is duplicated?

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Pull request closed without merging**